classwork/02-classwork.qmd

```diff
@@ -71,43 +71,13 @@ frog_test <- testing(frog_split)
 
 ## Exploratory data analysis for ML
```

Comment:
Should you just remove this header too? I think so, if nothing is going to be under it

Reply:
That's strange- on my branch I had changed it to a `#` section header, but it's not showing up in this diff 🤔  https://github.com/dgrtwo/workshops/blob/drob-deck-2/slides/02-data-budget.qmd#L220

Reply:
Once we merge I can see if it goes through. If it doesn't I'll open another! (I like the section header to separate this from the sampling discussion so far)